contracts/contracts/pausing/Pauser.sol

```diff
@@ -0,0 +1,232 @@
+// SPDX-License-Identifier: agpl-3.0
+pragma solidity ^0.8.0;
+
+import { Governable } from "../governance/Governable.sol";
+import { Initializable } from "../utils/Initializable.sol";
+import { Pausable } from "./Pausable.sol";
+
+contract Pauser is Initializable, Governable {
+    event ExpiryDurationChanged(uint256 _newExpiry);
+    event PausableChanged(address _newPausable);
+    event StrategistChanged(address _newStrategist);
+    event TempPaused(uint256 _expiry);
+    event Paused();
+    event Unpaused();
+    event Whitelisted(address _account);
+    event DeWhitelisted(address _account);
+
+    uint8 public constant UNPAUSED = 0;
```

Comment:
These there state values should probably be an `enum`, since that enforces that not just any number is okay here.

Reply:
Good point. Thanks for catching that. I'll update it.